src/components/common/Plotly.js

```diff
@@ -283,12 +283,32 @@ export default function Plotly() {
   let city1 = weatherCityData.cityWeather1;
   let city2 = weatherCityData.cityWeather2;
   let city3 = weatherCityData.cityWeather3;
+  let classes = ['main-temperature', 'main-temperature', 'main-temperature'];
+
+  function getTempsToSetClass(cityNum, num) {
+    console.log(cityNum, num);
+    num -= 1;
+    cityNum
+      ? // console.log(cityNum.imperial_main_temp)
+        cityNum.imperial_main_temp < 60
+        ? (classes[num] += ' tooCold')
+        : cityNum.imperial_main_temp > 80
+        ? (classes[num] += ' tooHot')
+        : (classes[num] += ' justRight')
+      : (classes[num] = ['main-temperature']);
+  }
+  getTempsToSetClass(city1, 1);
+  getTempsToSetClass(city2, 2);
+  getTempsToSetClass(city3, 3);
+  console.log(classes);
+  // city2 ? console.log(city2.imperial_main_temp) : console.log('object');
```

Comment:
Love this logic, I wonder how we can improve this later on down the road. 

Sidenote: Remove unused code/console.logs